src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/NotifyIconDesigner.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.ComponentModel.Design;
+
+namespace System.Windows.Forms.Design
+{
+    internal class NotifyIconDesigner : ComponentDesigner
+    {
+        private DesignerActionListCollection? _actionLists;
+
+        public override void InitializeNewComponent(IDictionary defaultValues)
+        {
+            base.InitializeNewComponent(defaultValues);
+            NotifyIcon icon = (NotifyIcon)Component;
+            icon.Visible = true;
+        }
+
+        public override DesignerActionListCollection ActionLists
+        {
+            get
+            {
+                _actionLists ??= new DesignerActionListCollection
+                {
+                    new NotifyIconActionList(this)
+                };
+
+                return _actionLists;
+            }
+        }
+    }
+
+    internal class NotifyIconActionList : DesignerActionList
+    {
+        private readonly NotifyIconDesigner _designer;
+        public NotifyIconActionList(NotifyIconDesigner designer) : base(designer.Component)
+        {
+            _designer = designer;
+        }
+
+        public void ChooseIcon()
+        {
+            EditorServiceContext.EditValue(_designer, Component, "Icon");
+        }
+
+        public override DesignerActionItemCollection GetSortedActionItems()
+        {
+            DesignerActionItemCollection items = new DesignerActionItemCollection
+            {
+                new DesignerActionMethodItem(this, "ChooseIcon", SR.ChooseIconDisplayName, true)
+            };
```

Comment:
we follow syntax and avoid duplicating type info.
```suggestion
            DesignerActionItemCollection items = new
            {
                new DesignerActionMethodItem(this, "ChooseIcon", SR.ChooseIconDisplayName, true)
            };
```

Reply:
I think this might not work, this will prompt compile error, to fix this error would need more code. I think it would not be as clean as it is.

Reply:
Then please use `var` instead of the explicit type